tests/unit/cbprintf/main.c

```diff
@@ -224,9 +230,11 @@ static inline bool prf_check(const char *expected,
 	};
 
 	const char *str = buf;
-	const char *sp = str;
-	int rc = match_pfx(&str);
+	const char *sp;
+	int rc;
 
+	sp = str;
```

Comment:
I don't see a huge value in leaving the values undefined at the definition, then assigning them immediately after.  But personal style, I guess.  I might automatically change them back if I have to edit this file again.